x-pack/legacy/plugins/security/index.js

```diff
@@ -127,7 +126,7 @@ export const security = (kibana) => new kibana.Plugin({
       isSystemAPIRequest: server.plugins.kibana.systemApi.isSystemApiRequest.bind(
         server.plugins.kibana.systemApi
       ),
-      cspRules: createCSPRuleString(config.get('csp.rules')),
+      cspRules: server.newPlatform.setup.core.http.csp.header,
```

Comment:
Now that CSP is in the NP, we don't need to have the Security plugin's legacy plugin "shim" it into the NP using `__legacyCompat.registerLegacyAPI`. Would you like us to address this in a separate PR? The changes don't look that extensive 🙌 https://github.com/kobelb/kibana/commit/feb29e0204c9352b2466749cbc085b501df14748

Reply:
I'll go ahead and incorporate since we are changing these anyway. Thank you for doing that!